Update GettingStarted.md (#171)
seL4/docs: GettingStarted.md

### Sources

Code and proofs are available on [GitHub](https://github.com/seL4), under standard

> [open-source licenses](https://github.com/seL4/seL4/tree/master/LICENSES), 

Reviewer: Maybe make this a bit more generic?
```suggestion
open-source licenses. Details can be found in the ``LICENSE.md`` file of each repository, 
e.g. [here](https://github.com/seL4/seL4/blob/master/LICENSE.md) for the seL4 kernel.
```

Reviewer: Bases on the remark from @lsf37 this could become:
```suggestion
open-source licenses. Details can be found in the ``LICENSES`` folder of each repository
(e.g. [here](https://github.com/seL4/seL4/tree/master/LICENSES) for the seL4 kernel) and
and in the [SPDX](https://spdx.dev) tags in each file
```

> e.g. [the sel4-kernel's](https://github.com/seL4/seL4/blob/master/LICENSE.md))

Reviewer: Seems there a bracket too much
```suggestion
e.g. [the sel4-kernel's](https://github.com/seL4/seL4/blob/master/LICENSE.md)
```
